roles/gateway/tasks/firewall.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: Make sure iptables isn't running
+  service:
+    name: iptables
+    state: stopped
+    enabled: false
+  ignore_errors: true
```

Comment:
The task returns an error `systemd could not find the requested service "iptables"` when the service isn't installed -- which it shouldn't be.  It's really just an extra step to make sure we're using firewalld and not iptables.